tests/ocean/test_compute_flow.py

```diff
@@ -217,3 +241,101 @@ def test_compute_multi_inputs():
         [compute_ddo, access_ddo],
         algo_ddo=algorithm_ddo,
     )
+
+
+def test_update_trusted_algorithms():
+    setup = Setup()
+
+    config = ConfigProvider.get_config()
+    ddo_address = get_contracts_addresses("ganache", config)[
+        MetadataContract.CONTRACT_NAME
+    ]
+    ddo_registry = MetadataContract(ddo_address)
+
+    # Setup algorithm meta to run raw algorithm
+    algorithm_ddo = get_registered_algorithm_ddo(
+        setup.publisher_ocean_instance, setup.publisher_wallet
+    )
+    # verify the ddo is available in Aquarius
+    _ = setup.publisher_ocean_instance.assets.resolve(algorithm_ddo.did)
+
+    # Dataset with compute service
+    compute_ddo = get_registered_ddo_with_compute_service(
+        setup.publisher_ocean_instance,
+        setup.publisher_wallet,
+        trusted_algorithms=[algorithm_ddo.did],
+    )
+    # verify the ddo is available in Aquarius
+    _ = setup.publisher_ocean_instance.assets.resolve(compute_ddo.did)
+    compute_ddo.update_trusted_algorithms(trusted_algorithms=[algorithm_ddo.did])
```

Comment:
The list of trusted algorithms is not just the algorithm did. Need to have the other hash values that are created in the helper function.